scripts/inference.py

```diff
@@ -18,15 +18,12 @@
 from typing import List, Optional
 import subprocess
 from transformers import AutoTokenizer
+import neural_speed
```

Comment:
This script is used for running executable files in C++, assuming no dependency on Python packages neural_speed

Reply:
Actually, when we build with setup.py file as in the README, neural_speed is there as package as u can see here:
![image](https://github.com/intel/neural-speed/assets/112881240/34bcdab4-55e6-4eb9-919d-ab8b84f49d50)